my-app/src/Components/Account.tsx

```diff
@@ -2,40 +2,25 @@ import React, { useState } from "react";
 import ToolTip from "./ToolTip";
 import AccountHover from "./AccountHover";
 import { useNavigate } from "react-router-dom";
+import { AccountProp } from "../TypesDefines/types";
 
-type Account = {
-  sidepanelOperation: (x: boolean) => void;
-  sidePanel: Boolean;
-};
-
-function Account({ sidepanelOperation, sidePanel }: Account) {
+function Account({ sidepanelMinimize, sidePanel }: AccountProp) {
+  let navigate = useNavigate();
   const [accounthov, setAccounthov] = useState<Boolean>(false);
-  window.addEventListener("click", () => m());
-  function m() {
+  window.addEventListener("click", () => clickHandle());
```

Comment:
We shouldn't attach an event listener directly inside the component if we do this, then new event listeners are added on every re-render. This can cause a memory leak, you should use `useEffect` for this and `removeEventListener` on the component unmount